calkit.yaml

```
@@ -12,14 +12,52 @@ notebooks:
   - path: modules/CEM/make_batch_csv.ipynb
     title: make-batch-csv
 environments:
-  decider:
+  decider-2:
     path: envs/calkit-environment.yml
     kind: conda
+dependencies:
+  - WLSACCESSID:
+      kind: env-var
+      notes: Gurobi WLS Access ID (API key)
+  - WLSSECRET:
+      kind: env-var
+      notes: Gurobi WLS Secret
+  - LICENSEID:
+      kind: env-var
+      notes: Gurobi license ID
+  - HSDS_API_KEY:
+      kind: env-var
+      notes: HSDS API key
+  # - gurobi_cl:
+  #     kind: app
+  #     notes: Gurobi command line interface
 pipeline:
   stages:
+    setup-hsds:
+      kind: shell-command
+      command: envsubst < .hscfg > ~/.hscfg
+      environment: decider-2
```

It is possible, though maybe risky, to run commands out in the system environment:

```suggestion
      environment: _system
```